collection-scripts/gather_services_status

```diff
@@ -214,6 +217,21 @@ get_glance_status() {
     run_bg ${BASH_ALIASES[os]} image task list '>' "$GLANCE_PATH"/task_list
 }
 
+# Watcher service gathering - task
+get_watcher_status() {
+    local WATCHER_PATH="$BASE_COLLECTION_PATH/ctlplane/watcher"
+    mkdir -p "$WATCHER_PATH"
+    run_bg ${BASH_ALIASES[os]} optimize goal list '>' "$WATCHER_PATH"/goal_list
+    run_bg ${BASH_ALIASES[os]} optimize strategy list '>' "$WATCHER_PATH"/strategy_list
+    run_bg ${BASH_ALIASES[os]} optimize audittemplate list '>' "$WATCHER_PATH"/audittemplate_list
+    run_bg ${BASH_ALIASES[os]} optimize audit list '>' "$WATCHER_PATH"/audit_list
+    run_bg ${BASH_ALIASES[os]} optimize actionplan list '>' "$WATCHER_PATH"/actionplan_list
+    run_bg ${BASH_ALIASES[os]} optimize action list '>' "$WATCHER_PATH"/action_list
+    run_bg ${BASH_ALIASES[os]} optimize service list '>' "$WATCHER_PATH"/watcher_service_list
+    run_bg ${BASH_ALIASES[os]} optimize datamodel list '>' "$WATCHER_PATH"/watcher_datamodel_list
```

Comment:
this is not correct we shoudl not be collecting the datamodel actions, audits or action plans

the goals and stragies would only hcange if they added there own plugins for thos so the only thing i thikn we shoudl likely keep is the service list

```suggestion
    run_bg ${BASH_ALIASES[os]} optimize service list '>' "$WATCHER_PATH"/watcher_service_list
```

if you look at the nova version you can see we intentiolly do not collelct the flavor or instances as we are not ment to collect the workload info via must gather

collection the action plans, actions autit templates and audits re all "workload info" in the context of watcher.

i maybe coudl see collectin the list of audittempelates but honeslty i dotn think that will be useful for debuging.

Reply:
I agree that probably the only useful part will be the service list and maybe audits (i.e. to see the parameters for questions like "my audit X is not doing what I'd expect) although for those case it will probably easier to ask for specific commands.

We can also keep the list of goals and strategies just to check that the deployment of watcher went fine.

Said that, I'd say the only bad side of it is extra space usage, right?

Reply:
the goals and stragies i think is fine

the probelm with the workload information is w eshoudl not be collecting data that might be sensivite and things liek the parmaters pass to an audit may be.

certainly i know we have to be careful with collecting ips and hostnames in general, the service list i think is ok but i think the datamodel is too far at least by default.

we could support it via a flag but i strongly suspect we would be much better off requesting this info explicit if necessary. if that happen often we can expand the list but for now lets keep this pretty simple and short.